### PR TITLE
 Forgot to remove the "Adjudicate Next" button - Oops!

### PR DESCRIPTION
**Pull request by ChidiNweze**
When we were showing the website to Eric, I realized I forgot to remove the "adjudicate next" button in the admin view. I just quickly fixed this because it was bugging me. Sorry for the small PR 😞.

### CODE REVIEW

File: frontend/src/components/adjudications/AdjudicationSection.jsx [hunk 1]
```diff
@@ -65,14 +71,17 @@ export default function AdjudicationsSection(props) {
     history.push(`/events/${eventId}/adjudications/performance/${nextPerformance.id}`);
   };
 
+  var adjudicateNextButton = <Button variant="outlined" color="primary" onClick={goToNextPerformance}>Adjudicate Next</Button>;
+  if(isAdmin) {
```
Comment (brandonleung) [inline frontend/src/components/adjudications/AdjudicationSection.jsx:75]: you can handle this with a ternary expression

`var adjudicateNext = isAdmin ? <></> : <Button variant ... > Adjudicate Next </Button>`

File: frontend/src/components/adjudications/AdjudicationTableRow.jsx [hunk 2]
```diff
@@ -8,18 +8,25 @@ import IconButton from "@material-ui/core/IconButton";
...
+  state = {adjudication: {}, isEditMode: false, technicalMark: {}, artisticMark: {}, isAdmin: true, judges: {}}; //hardcoded admin value
 
   componentDidMount() {
     getAdjudicationByPerformanceAndJudge(this.props.id, 1) //hradcoded tablet_id
     .then(({data}) => {
       this.setState({ adjudication: data, technicalMark: data.technicalMark, artisticMark: data.artisticMark});
+    }); 
+    getJudgesWhoCompletedAdjudication(parseInt(this.props.id))
+    .then(({data}) => {
+      this.setState({judges: data});
```
Comment (brandonleung) [inline frontend/src/components/adjudications/AdjudicationTableRow.jsx:25]: does `data` contain all the http boilerplate? like status code and headers and all that? 

by directly associating the api response with our state we might get some unwanted behaviours with future development

**Review by brandonleung — CHANGES_REQUESTED** [review]
small comments

File: frontend/src/components/adjudications/AdjudicationTableRow.jsx (hunk 2 above)
Comment (ChidiNweze) [reply frontend/src/components/adjudications/AdjudicationTableRow.jsx:25]: Data is just an object containing the tablet_ids of the judges who completed that adjudication. I'm pretty sure it only contains integers.

**Review by brandonleung — APPROVED** [review]
Nice!

**Pull request merged**